.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E402
+ignore = E402,W503
```

Comment:
For context, disabled due to https://www.flake8rules.com/rules/W503.html being considered an antipattern. It would also conflict with formatters like black. Updating packages is outside the scope of this PR, hence lint ignored.